common/web/keyboard-processor/src/keyboards/keyboard.ts

```diff
@@ -558,19 +585,19 @@ export default class Keyboard {
    * @return      {number}                key code > 255 on success, or 0 if not found
    */
   getVKDictionaryCode(keyName: string) {
+    const dict = this.scriptObject['VKDictionary'] || {} as KeyboardObject['VKDictionary'];
```

Comment:
Why did you hoist this outside the if statement? It is only used within the if statement.

Reply:
This allows us to get proper typing while initializing as `const`, rather than `let`.

In retrospect... why didn't I use it in line 603?  It'd be perfectly valid, and would be an outside-of-if use that should resolve your concern.

Reply:
Yes, just move it back into the `if` and simplify:

```
const dict = {} as KeyboardObject['VKDictionary'];
```